tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """
+    When collecting tests look at the test path
+    and if it includes the name of a type of ceph
+    daemon in it, tag the test with the appropriate
+    marker. This allows for us to filter and test
+    by specific daemon type.
+
+    For example, all tests that live in a file at
+    tests/mon/test_mons.py would be tagged with
+    a 'mons' marker.
+    """
+    for item in items:
+        test_path = item.location[0]
```

Comment:
Hi @andrewschoen, what item.location[0] contains? First directory in which the test file belongs to?  We discussed to have structure starting with teams folders in QE. And we merged PR #61 which implements pytest for us and the basic config we had in old runner to pytest. So if this structure is needed from engineering can this been discussed on our regular meeting? Thanks